unblob/extractor.py

```diff
@@ -85,12 +91,24 @@ def fix_symlink(path: Path, outdir: Path, task_result: TaskResult) -> Path:
 
 
 def fix_extracted_directory(outdir: Path, task_result: TaskResult):
+    def _fix_extracted_directory(directory: Path, task_result: TaskResult):
```

Comment:
```suggestion
    def _fix_extracted_directory(directory: Path):
```

`task_result` is accessible from outer scope.